tests/ISADotnet.Tests.NetCore/ISADotnet/API/UpdateTests.fs

```diff
@@ -0,0 +1,252 @@
+﻿module UpdateTests
```

Comment:
Module name APITests would be more appropriate

Reply:
The unit tests i added are only to test the functionality written in Update.fs. If you want to add API tests to this .fs in the future i will of course change the name.

Reply:
Yes, other API tests will surely follow and I think it might be more comprehensible to group them in a APITests module